### ISSUE TITLE
Question: What is happening to version 1.7?

### ISSUE THREAD

**Issue by Sebastian1989101** [praeclarum/sqlite-net#940]
More and more nuget packages depending on SQLite raw v2 which cannot be updated until this library also has a version for it. But the last update to it is now 7 month old.

**Comment by sk1llsh0t** [comment]
I agree, compatibility issues are starting to creep in lately. any update on this?

**Comment by emiliano84** [comment]
I agree, also I don't see any commit on this repo related to version 1.7

**Comment by praeclarum** [comment]
Sorry about that. I'm prepping a release for today!

**Comment by emiliano84** [comment]
@praeclarum great, thanks! I think you can close this issue :)
